scripts/asr_language_modeling/ngram_lm/train_kenlm.py

```diff
@@ -23,12 +23,11 @@
 # You need to install the KenLM library and also the beam search decoders to use this feature. Please refer
 # to 'scripts/ngram_lm/install_beamsearch_decoders.sh' on how to install them.
 #
-# USAGE: python train_kenlm.py --nemo_model_file <path to the .nemo file of the model> \
-#                              --train_file <path to the training text or JSON manifest file \
+# USAGE: python train_kenlm.py --tokenizer_model_file <path to the .nemo file of the model> \
```

Comment:
The nemo docs need to get updated with these changes.

Reply:
We have some documentation on LM for ASR here:
https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/asr/asr_language_modeling.html

The name of the parameters like tokenizer_model_file need to get updated in those docs.